trl/trainer/dpo_trainer.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import warnings
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+from datasets import Dataset
+from transformers import DataCollator, PreTrainedModel, PreTrainedTokenizerBase, Trainer, TrainingArguments
+from transformers.trainer_callback import TrainerCallback
+from transformers.trainer_pt_utils import nested_detach
+from transformers.trainer_utils import EvalPrediction
+
+from ..core import logprobs_from_logits, masked_mean
+from ..import_utils import is_peft_available
+from .utils import DPODataCollatorWithPadding
+
+
+if is_peft_available():
+    from peft import get_peft_model
+
+
+def compute_accuracy(eval_pred) -> Dict[str, float]:
```

Comment:
I think this method is copied from: https://github.com/lvwerra/trl/blob/7705daa672f9264d7b5b789b6a1fd6b1cff03a58/trl/trainer/reward_trainer.py#L34 
Can you remove this method and move the method that is on the RewardTrainer to `trainer/utils` and import it from there?